core/token-pooling/redis-token-persistence.integration.js

```diff
@@ -84,8 +75,8 @@ describe('Redis token persistence', function() {
         await persistence.initialize()
         await persistence.noteTokenAdded(newToken)
 
-        const savedTokens = await lrange(key, 0, -1)
```

Comment:
i'm not familiar enough with these redis clients to follow some of the changes like this. assume it's all correct but would need to read up on those apis to offer any hepful feedback

Reply:
It's better to have more eyes, though we don't necessarily need to block on it. The worst thing that happens is we have bugs that need fixing. I have the tokens backed up so we can roll back the data to this point if need be.

Reply:
Works for me. Will give this a quick final look later today and approve